Uplift/AppDelegate.swift

```diff
@@ -51,15 +53,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         }
     }
-    
+
     func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
         return GIDSignIn.sharedInstance().handle(url as URL?)
     }
+
+    // MARK: - Onboarding
+    private func setupOnboardingViewController() {
+        self.window?.rootViewController = UIViewController()
+
+        let gyms: [String] = ["Helen Newman", "Appel", "Teagle", "Noyes"]
+        let classes: [GymClassInstance] = [
+                GymClassInstance(classDescription: "", classDetailId: ClassIds.yogaVinyasa, className: "Yoga Vinyasa", duration: 0, endTime: Date(), gymId: "", imageURL: URL(fileURLWithPath: ""), instructor: "", isCancelled: false, location: "", startTime: Date(), tags: []),
+                GymClassInstance(classDescription: "", classDetailId: ClassIds.CURowShockwave, className: "CU Row (Shockwave)", duration: 0, endTime: Date(), gymId: "", imageURL: URL(fileURLWithPath: ""), instructor: "", isCancelled: false, location: "", startTime: Date(), tags: []),
+                GymClassInstance(classDescription: "", classDetailId: ClassIds.zumba, className: "Zumba", duration: 0, endTime: Date(), gymId: "", imageURL: URL(fileURLWithPath: ""), instructor: "", isCancelled: false, location: "", startTime: Date(), tags: []),
+                GymClassInstance(classDescription: "", classDetailId: ClassIds.musclePump, className: "Muscle Pump", duration: 0, endTime: Date(), gymId: "", imageURL: URL(fileURLWithPath: ""), instructor: "", isCancelled: false, location: "", startTime: Date(), tags: [])
+            ]
+
+        self.window?.rootViewController = OnboardingViewController(gymNames: gyms, classes: classes)
```

Comment:
i feel like we should make this function return `OnboardingViewController` and then we set the `rootViewController` on line 32.  otherwise, its unclear whether or not this function actually handles setting the `rootViewController`